docs/content.zh/docs/dev/table/sql/create.md

```diff
@@ -33,6 +33,7 @@ CREATE 语句用于向当前或指定的 [Catalog]({{< ref "docs/dev/table/catal
 目前 Flink SQL 支持下列 CREATE 语句：
 
 - CREATE TABLE
+- CREATE TABLE [USING]
```

Comment:
Why `CREATE TABLE [USING]`?

Reply:
`CREATE TABLE [USING]` is a reference to the[ databricks spark docs](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-ddl-create-table-using.html), I'll adjust it to `[CREATE OR] REPLACE TABLE` to be more intuitive